realio/chain.json

```diff
@@ -0,0 +1,124 @@
+{
+    "$schema": "../chain.schema.json",
+    "chain_name": "realio",
+    "status": "live",
+    "network_type": "mainnet",
+    "website": "https://realio.network/",
+    "pretty_name": "Realio",
+    "chain_id": "realionetwork_3301-1",
+    "bech32_prefix": "rio",
+    "node_home": "$HOME/.realio-network",
+    "daemon_name": "realio-networkd",
+    "key_algos": [
+      "ethsecp256k1"
+    ],
+    "slip44": 60,
+    "fees": {
+      "fee_tokens": [
+        {
+          "denom": "ario",
+          "fixed_min_gas_price": 1000000000,
+          "low_gas_price": 4000000000,
+          "average_gas_price": 5000000000,
+          "high_gas_price": 8000000000
+        }
+      ]
+    },
+    "staking": {
+      "staking_tokens": [
+        {
+          "denom": "ario"
+        },
+        {
+          "denom": "arst"
```

Comment:
just want to double check, because this would be a first. Both these tokens are staking tokens for securing the network?

Reply:
yes , realio have two native . Delegators can stake rio and rst